### PR TITLE
Week 9 Email

### PR DESCRIPTION
**Pull request by jrtechs**
Drafted week 9 email.

### CODE REVIEW

File: announcements/_posts/2019-10-25-w09-pi-fest.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+Hey Ritluggers,
+
+This week we will be exploring all the wonderful things you can do with Raspberry Pis.
+We will go over some Pi 101 things and then dive into demoing some of our favorite Raspberry pi projects.
+If anyone in the club has a Raspberry Pi, feel free to bring it with you to the club this Friday.
+
+Tomorrow (Thursday) there will be a "pizza social" during FOSS hours where we are meeting and greeting FOSS students faculty and alums.
+Everyone is welcome to come, FOSS hours are held weekly on Thursdays from 5-7 PM in Magic Center 3190.
```
Comment (Tjzabel) [inline announcements/_posts/2019-10-25-w09-pi-fest.md]: Everyone is welcome to come: FOSS Hours are held....

File: announcements/_posts/2019-10-25-w09-pi-fest.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+location: "GOL-2620 (Med. DB Lab)"
+---
+
+Hey Ritluggers,
+
+This week we will be exploring all the wonderful things you can do with Raspberry Pis.
+We will go over some Pi 101 things and then dive into demoing some of our favorite Raspberry pi projects.
+If anyone in the club has a Raspberry Pi, feel free to bring it with you to the club this Friday.
+
+Tomorrow (Thursday) there will be a "pizza social" during FOSS hours where we are meeting and greeting FOSS students faculty and alums.
```
Comment (Tjzabel) [inline announcements/_posts/2019-10-25-w09-pi-fest.md]: "...where there will be a meet-and-greet with other FOSS students, faculty, and alums."

File: announcements/_posts/2019-10-25-w09-pi-fest.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+Everyone is welcome to come, FOSS hours are held weekly on Thursdays from 5-7 PM in Magic Center 3190.
+This weekend, FOSS at Magic is hosting The Future Is Open conference which is the first-ever open-source conference at RIT.
```
Comment (Tjzabel) [inline announcements/_posts/2019-10-25-w09-pi-fest.md]: * "FOSS@MAGIC": Proper way of writing it
* "_The Future Is Open_": Maybe italicize to make the title jump out more

File: announcements/_posts/2019-10-25-w09-pi-fest.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+RITlug will be helping host three events/talks, there will be **more free pizza**, and there are dozens of guest speakers coming.
+This event is all day Saturday (10 AM-5 PM) in the Magic center -- we got the entire building reserved for this event!
```
Comment (Tjzabel) [inline announcements/_posts/2019-10-25-w09-pi-fest.md]: "MAGIC center"

File: announcements/_posts/2019-10-25-w09-pi-fest.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+I hope to see you all at this week's meeting!
+
+- Jeffery Russell (@jrtechs)
+
+
+**Upcoming Events:**
+
+* [The Future Is Open](https://fossrit.github.io/events/2019/10/26/the-future-is-open/): October 26th
+* [Rochester Maker Faire](https://rochester.makerfaire.com/): November 23rd
+* FOSS Hours: Thursdays 5:00-7:00pm Magic Center 3190
```
Comment (Tjzabel) [inline announcements/_posts/2019-10-25-w09-pi-fest.md]: "MAGIC Center"

**Comment by jrtechs** [comment]
Groovy thanks @Tjzabel

**Pull request merged**